### ISSUE TITLE
Support SGE 

### ISSUE THREAD

**Issue by BTopcuoglu** [SchlossLab/mikropml-snakemake-workflow#10]
We have a working solution locally as well as Slurm. Let's get it running on SGE as well.

**Comment by kelly-sovacool** [comment]
SGE the job scheduler, right? You'll probably just need to create a profile in `config/sge/config.yaml`, add an example submission script in `code/submit_sge.sh`, and _maybe_ edit `config/cluster.json` if needed.

**Comment by BTopcuoglu** [comment]
Yes, SGE the scheduler - @ykefella and I are working on this and will update y'all on the progress.

**Comment by kelly-sovacool** [comment]
@BTopcuoglu You might find it helpful to fork & modify [this](https://github.com/kelly-sovacool/snakemake_hpc_mwe) to get a simpler Snakemake workflow running with SGE
